Symfony/CS/Tokenizer/Token.php

```diff
@@ -99,9 +100,22 @@ public function equals($other)
         $selfPrototype = $this->getPrototype();
 
         foreach ($otherPrototype as $key => $val) {
-            if ($selfPrototype[$key] !== $val) {
+            // Make sure the token has such key
```

Comment:
Will fix. Could you please point me to such rules for the comments? I'd be happy to read more about them.=